internal/cli/cmd/connectivity.go

```diff
@@ -119,6 +119,7 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().StringVar(&params.AssumeCiliumVersion, "assume-cilium-version", "", "Assume Cilium version for connectivity tests")
 	cmd.Flags().BoolVarP(&params.Verbose, "verbose", "v", false, "Show informational messages and don't buffer any lines")
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
+	cmd.Flags().BoolVarP(&params.DenyPolicy, "deny-policy", "", true, "Run tests with deny policy")
```

Comment:
Do we really need this specific flag? Couldn't we just use something like `'!/.*-deny' instead to skip deny policy tests?

Reply:
do you know how to skip multiple patterns ? I tried a few cases but failed, so just add the flag for easier control 🤔

Reply:
It looks like `--test` is [defined](https://github.com/cilium/cilium-cli/blob/374d13759ea397c8c28bd840b57254b46d1e577d/internal/cli/cmd/connectivity.go#L116) as a [`StringSliceVar`](https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar), so either multiple invocations (i.e. `--test '!/*-deny' --test '!/pod-to-.*-nodeport'`) or a comma-separate `--test` invocation (i.e. `--test '!/*-deny,!/pod-to-.*-nodeport'`, like e.g. [here](https://github.com/cilium/cilium-cli/blob/374d13759ea397c8c28bd840b57254b46d1e577d/.github/in-cluster-test-scripts/eks-tunnel.sh#L32)) should work. If it doesn't, then we should fix/extend `--test` parsing/handling rather than introducing new flags for each special case.